assignment/project/internal/service/favourite/service.go

```diff
@@ -0,0 +1,61 @@
+package favourite
+
+import (
+	ct "golang-project/internal/contract"
+	repo "golang-project/internal/repository"
+	svc "golang-project/internal/service"
+	"golang-project/static"
+)
+
+// service represents the implementation of service.Favourite
+type service struct {
+	favouriteRepo repo.Favourite
+	userRepo      repo.User
+}
+
+// NewService returns a new implementation of service.Favourite
+func NewService(favouriteRepo repo.Favourite, userRepo repo.User) svc.Favourite {
+	return &service{
+		favouriteRepo: favouriteRepo,
+		userRepo:      userRepo,
+	}
+}
+
+// ListFollowers returns a list of all bloggers that a user is following
+func (s *service) ListFollowers(userID int) (*ct.ListProfileResponse, error) {
+	// Check if user exists
+	_, err := s.userRepo.Read(userID)
+	if err != nil {
```

Comment:
cái này chỉ nên trả về 1 lỗi là user not found thôi
ko nên phân chia thành 2 lỗi như này, nó giảm tính bảo mật